CHANGELOG.md

```diff
@@ -1,5 +1,128 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## v5.0.0-alpha.24
+
+<!-- generated comparing v5.0.0-alpha.23..next -->
+
+_Jan 25, 2021_
+
+Big thanks to the 21 contributors who made this release possible: @eps1lon, @mbrookes, @mnajdova, @greguintow, @praveenkumar-kalidass, @oliviertassinari, @lukaselmer, @povilass, @eladmotola, @kutnickclose, @artola, @vinyldarkscratch, @KarimMokhtar, @mxsph, @tonysepia, @d3mp, @CyanoFresh, @xs9627, @vicasas, @TimonPllkrn, @duganbrett
```

Comment:
If we want to list all contributors, would suggest doing it at the end of this section. It takes too much place at the start. Also, we have the contributors anyway next to each PR, so I am not sure if this brings more value...

Reply:
Done